.github/dependabot.yml

```diff
@@ -6,17 +6,6 @@
 version: 2
 updates:
   # regular updates for our established packages
-  - package-ecosystem: "npm"
-    directory: "/" # Location of package manifests
-    schedule:
-      interval: "daily"
-    target-branch: "master"
-    ignore:
-      - dependency-name: "*"
-        update-types: ["version-update:semver-major"]
-    labels: 
-      - "minor"
-  # major updates:
   - package-ecosystem: "npm"
     directory: "/" # Location of package manifests
     schedule:
```

Comment:
Should we just get rid of the package version locks here even if each generated PR has to remain open for a while?

Reply:
Sure!